crates/core/src/values/wrap_content.rs

```
@@ -0,0 +1,16 @@
+use torin::wrap_content::WrapContent;
+
+use crate::parsing::{
+    Parse,
+    ParseError,
+};
+
+impl Parse for WrapContent {
+    fn parse(value: &str) -> Result<Self, ParseError> {
+        Ok(match value {
+            "no-wrap" => WrapContent::NoWrap,
+            "wrap" => WrapContent::Wrap,
+            _ => WrapContent::NoWrap,
```

this is incorrect, you need to return a parse error if the value type is incorrect